### PR TITLE
update table's selection function

### PR DESCRIPTION
**Pull request by jonkiky**

### CODE REVIEW

File: src/bento/caseDetailsData.js [hunk 1]
```diff
@@ -176,6 +178,7 @@ export const table2 = {
       dataField: 'uuid',
       header: 'UUID',
       display: false,
+      primary: true,
```
Comment (AjayDoddapaneni) [inline src/bento/caseDetailsData.js:181]: why didn't we use this instead of primary key index?

Comment (jonkiky) [reply src/bento/caseDetailsData.js:181]: Column setting will be redefined from Bento-tools's function getColumns. 
The attribute primary was not included in the getColumns function.  Even set this attribute here, will not be used.  Change the getColumns from bento-tools is challenging at the point and we need some design thinking of how to improve the getColumns and getOption functions to be more flexible.

**Review by AjayDoddapaneni — APPROVED** [review]
Okay we need to create a ticket to explore more on this

**Review by AjayDoddapaneni — APPROVED** [review]

**Comment by skiran86** [comment]
created a ticket in the backlog to investigate for improvement https://tracker.nci.nih.gov/browse/BENTO-1025

**Review by skiran86 — APPROVED** [review]

**Pull request merged**